snippets/_evm-sidechain-disclaimer.md

```diff
@@ -1,4 +0,0 @@
-{% admonition type="info" name="Attention" %}
```

Comment:
Why did we remove this snippet? The information here is still accurate, isn't it?

Reply:
Despite being partially accurate, we have removed it in order to prepare the docs for the upcomming Testnet and Mainnet launch